games/index.html

```diff
@@ -24,7 +25,7 @@
         <!--Navigation bar-->
         <ul>
             <li><a href=".." class="home">Home</a></li>
-            <li><a>Games</a></li>
+            <li><a class="games">Games</a></li>
```

Comment:
Use / instead so it always points to root, no matter where the file is


```suggestion
            <li><a href="/" class="home">Home</a></li>
```